lib/apiservers/service/swagger.json

```diff
@@ -830,6 +830,12 @@
     }
   },
   "parameters": {
+    "session-ticket": {
```

Comment:
This doesn't seem to be used.

Reply:
Hmm yeah, it looks like the ticket need only be used in the session security definition. I'll remove and re-test.